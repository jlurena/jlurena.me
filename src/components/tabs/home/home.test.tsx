import { render } from '@testing-library/react'
import Home from './home'

describe('Home', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Home />)
    expect(asFragment()).toMatchSnapshot()
  })
})
