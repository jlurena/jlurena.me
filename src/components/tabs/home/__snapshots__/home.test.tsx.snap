// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home > matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="_content_a33fb6"
  >
    <img
      alt="Picture of Jean Luis Urena"
      class="_picture_a33fb6"
      height="200"
      src="/src/assets/images/jean/webslidepic1.jpeg"
      width="200"
    />
    <div
      class="_introduction_a33fb6"
    >
      <h1>
        Hello, my name is
      </h1>
      <div
        class="_name_a33fb6"
      >
        Jean Luis Ure√±a
      </div>
    </div>
    <div
      class="_wordsCarousel_a33fb6"
    >
      I am a
      <div>
        <span
          class="_word_6729c1"
        >
          Leader
        </span>
        <span
          class="_word_6729c1"
        >
          Software Engineer
        </span>
        <span
          class="_word_6729c1"
        >
          Student
        </span>
        <span
          class="_word_6729c1"
        >
          Car Enthusiast
        </span>
        <span
          class="_word_6729c1"
        >
          Hiker
        </span>
        <span
          class="_word_6729c1"
        >
          Coffee Fanatic
        </span>
      </div>
    </div>
    <div
      class="_contact_a33fb6"
    >
      <a
        class="reverse"
        href="mailto:eljean@live.com?subject=&body="
      >
        moc.evil@
        <span />
        naejle
      </a>
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-paper-plane "
        data-icon="paper-plane"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M498.1 5.6c10.1 7 15.4 19.1 13.5 31.2l-64 416c-1.5 9.7-7.4 18.2-16 23s-18.9 5.4-28 1.6L284 427.7l-68.5 74.1c-8.9 9.7-22.9 12.9-35.2 8.1S160 493.2 160 480l0-83.6c0-4 1.5-7.8 4.2-10.8L331.8 202.8c5.8-6.3 5.6-16-.4-22s-15.7-6.4-22-.7L106 360.8 17.7 316.6C7.1 311.3 .3 300.7 0 288.9s5.9-22.8 16.1-28.7l448-256c10.7-6.1 23.9-5.5 34 1.4z"
          fill="currentColor"
        />
      </svg>
    </div>
  </div>
</DocumentFragment>
`;
